### PR TITLE
Release v2.0.4+rev2

### PR DESCRIPTION
**Pull request by floion**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Change log
 -----------
 
+# v2.0.4+rev2 - 2017-05-28
```
Comment (agherzan) [inline CHANGELOG.md:4]: Why do we have two releases in this PR? We could include the uboot change in rev1 too.

Comment (floion) [reply CHANGELOG.md:4]: The uboot change does not make sense until meta-resin is updated to 2.0.4

Comment (agherzan) [reply CHANGELOG.md:4]: Sure. But you could have one release with both meta-resin bump and uboot adaptions. That's what I would do.

**Review by agherzan — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (floion) [reply CHANGELOG.md:4]: Yeah, but until now we did a release when we bumped meta-resin. I thought I'd follow the same convention

Comment (agherzan) [reply CHANGELOG.md:4]: I think even in the past we did a release in every meta-resin bump but we included the modifications needed for that release. Anyway - it's more of a suggestion.

**Pull request merged**